### ISSUE TITLE
bug: Navigating to a page that has been been loaded before crashes app

### ISSUE THREAD

**Issue by ebk46** [ionic-team/ionic#19777]
# Bug Report

**Ionic version:**
[x] **4.11.2 (React)**

**Current behavior:**
Navigating to a page that has been loaded before breaks the app 100% of the time.

**Expected behavior:**
The page should render when the app navigates to it.

**Steps to reproduce:**
See StackBlitz below. Very easy to reproduce:
1. Assuming you are starting from Page 1, use the button on the bottom to go to Page 2
2. Use the button on the bottom of Page 2 to go back to Page 1.
3. The app will crash.

You can also start from Page 2 and it will crash when you return to Page 2.

**Related code:**
https://stackblitz.com/edit/ionic-v4-11-2-react-call-stack

**Other information:**
Error seems to be from `RouteManager.removeOrphanedViews`

**Ionic info:**
```
Ionic:

   Ionic CLI       : 5.4.4 (/Users/evan/.config/yarn/global/node_modules/ionic)
   Ionic Framework : @ionic/react 4.11.2

Capacitor:

   Capacitor CLI   : 1.3.0
   @capacitor/core : 1.3.0

Utility:

   cordova-res : 0.6.0 (update available: 0.8.1)
   native-run  : not installed

System:

   NodeJS : v12.10.0 (/usr/local/Cellar/node/12.10.0/bin/node)
   npm    : 6.11.3
   OS     : macOS High Sierra
```

**Comment by liamdebeasi** [comment]
Thanks for the issue. This bug has already been fixed and should be available in the next release of Ionic.

You can try it out with this development build of Ionic: `npm i @ionic/react@4.12.0-dev.201910281628.f70e71a @ionic/react-router@4.12.0-dev.201910281628.f70e71a`


Thanks!

**Comment by OoDeLally** [comment]
Same issue here. Works with @liamdebeasi 's proposition.

**Comment by Rinbo** [comment]
May I ask when the next release will be out?

**Comment by liamdebeasi** [comment]
This was fixed in Ionic React 4.11.3.
